financial_calculations/bond_cash_flows.py

```diff
@@ -54,13 +54,13 @@ def create_semi_bond_cash_flows(effective_date, balance, coupon, maturity_years)
         raise ValueError("Day of the month should not be greater than 28 to avoid end-of-month issues.")
 
     # Calculate the number of payments (semiannual payments)
-    num_payments = int(FRAC_OF_YEAR * maturity_years)
+    num_payments = int(PMTS_PER_YEAR * maturity_years)
```

Comment:
this is a nit, but you don't have to cast to an int. at least I am pretty sure.  these should both be ints already